### PR TITLE
Updates to FAM actions utility functions

### PR DESCRIPTION
**Pull request by badvision**
- Complete rewrite of how ActionManager manages resource resolvers
- Identify assets with missing or outdated renditions
- Activate content to a list of publishers in round-robin manner (parallelizes publishing where available)
- Translation function to apply actions against renditions (useful for activating or deleting specific renditions)

### CODE REVIEW

File: bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java [hunk 1]
```diff
@@ -131,25 +147,26 @@ public void accept(ResourceResolver r) throws Exception {
...
-                        t.commit();
-                        t.close();
-                    } catch (PersistenceException ex) {
-                        LOG.error("Persistence exception closing session pool", ex);
+        Runnable r = new Runnable() {
+            @Override
+            public void run() {
+                while (!isComplete()) {
+                    try {                    
+                        Thread.sleep(100);
```
Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java]: Curious, whats the significance of this magic number?

File: bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java [hunk 2]
```diff
@@ -298,10 +287,13 @@ public CompositeData getStatistics() throws OpenDataException {
     public void closeAllResolvers() {
         if (!resolvers.isEmpty()) {
             for (ReusableResolver resolver : resolvers) {
-                resolver.getResolver().close();
+                if (resolver.getResolver().isLive()) {
```
Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java]: This should be closed regardless of `.isLive()` check .. .javadocs are misleading. see @cziegeler note on the sling lists: 

http://apache-sling.73963.n3.nabble.com/Using-the-resource-resolver-td4061854.html

Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java]: Will fix this when i use it.

**Comment by badvision** [comment]
Seriously?   Wow.  I thought that isLive() indicates if the session is open and false = closed.  Yeah, please change it for sure if you think it will be ok.

File: bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java (hunk 1 above)
Comment (badvision) [inline bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java]: 100ms?  Nothing terribly significant, I just want to make sure that we're not in a tight wait-loop while other threads complete work and waiting several seconds for that to happen is almost certain.

**Pull request merged**